src/pages/career/Benefits.tsx

```diff
@@ -96,9 +92,9 @@ const Benefits: React.FC = () => {
 
         <BlockSmall className='is-pulled-right'>
           <span>Follow us on:</span>
-          <a target='_blank' href={'https://www.linkedin.com/company/bright-inventions/'}><img src={imgLinkedIn} alt={'LinkedIn'} /></a>
-          <a target='_blank' href={'https://www.facebook.com/bright.inventions/'}><img src={imgFacebook} alt={'Facebook'} /></a>
-          <a target='_blank' href={'https://www.instagram.com/bright_inventions/'}><img src={imgPinterest} alt={'Pinterest'} /></a>
+          <a target='_blank' href='https://www.linkedin.com/company/bright-inventions/'><img src='/images/social/facebook.svg' alt='LinkedIn' /></a>
```

Comment:
Could you please elborate why we're not using GatsbyImage here as well? Is it because of svg?

Reply:
actually no.. I just didn't spot those 😅 
created PR for it and 2 bugs for map keys